### ISSUE TITLE
Files crashed while opening a folder

### ISSUE THREAD

**Issue by Firnin** [files-community/Files#9136]
### Description

I wanted to open a folder on my local hard drive and Files crashed.
Just to be clear, this happened only once and I wasn't able to reproduce the error. The steps I listed down below were the steps I was doing at the time Files crashed

### Steps To Reproduce

1. Navigate to C:\
2. Open any folder with a deeper folder structure in it
3. Open a folder
4. It crashes

### Expected behavior

I expected Files to load the folder and not to crash.

### Files Version

Version: 2.1.15.0 OS Version: 10.0.22000.652

### Windows Version

Win 11 Pro 21H2

### Relevant Assets/Logs

[debug.log](https://github.com/files-community/Files/files/8659486/debug.log)
[debug_fulltrust.log](https://github.com/files-community/Files/files/8659487/debug_fulltrust.log)

**Comment by gave92** [comment]
Thanks for the report. Closing this as this error is sadly common and already tracked in other issues.
